src/update_handler.c

```diff
@@ -584,6 +584,40 @@ static gboolean vfat_format_slot(RaucSlot *dest_slot, GError **error)
 	return res;
 }
 
+static gboolean flash_write_slot(const gchar *image, const gchar *device, GError **error)
+{
+	g_autoptr(GSubprocess) sproc = NULL;
+	GError *ierror = NULL;
+	gboolean res = FALSE;
+	g_autoptr(GPtrArray) args = g_ptr_array_new_full(5, g_free);
+
+	g_ptr_array_add(args, g_strdup("flashcp"));
+	g_ptr_array_add(args, g_strdup(image));
+	g_ptr_array_add(args, g_strdup(device));
+	g_ptr_array_add(args, NULL);
```

Comment:
Shoudn't this use `--erase-all`? For other slots we make sure control the full resulting contents.

Reply:
This already writes the slot, I wonder if I can simply reuse the `nand_format_slot` handler to erase the device before flashing the image.

Reply:
Did that, please re-review.